draft-ietf-masque-connect-ip.md

```diff
@@ -1046,6 +1048,33 @@ References:
 : This document
 {: spacing="compact"}
 
+## Creation of the MASQUE URI Suffixes Registry {#iana-suffix}
+
+This document requests that IANA create a new "MASQUE URI Suffixes" registry
+maintained at IANA_URL_TBD. This new registry contains three columns:
+
+Path Segment:
+: An ASCII string containing only characters allowed in tokens; see
+{{Section 5.6.2 of HTTP}}. Entries in this registry MUST all have distinct
+entries in this column.
+
+Description:
+: A description of the entry.
+
+Reference:
+: An optional reference defining the use of the entry.
+
+The registration policy for this registry is Expert Review; see
+{{Section 4.5 of !IANA-POLICY=RFC8126}}.
```

Comment:
I am fine with Expert review. However, should one of the criteria be a specification that the expert can look at? And I assume the more general reason for expert review is to prevent malicious use of simply filling up the registry with garbage?

Reply:
Actually I would go for IETF Review. If we expect IETF RFCs for all usages we don't need experts.

Reply:
I think requiring a specification often leads to folks squatting on the namespace instead of registering so I'd prefer a low bar. I avoided "First Come first served" to prevent spam entries. Let's bikeshed the registration policy during WGLC, Mirja please file a separate issue if you feel strongly